doc/release-notes/master.txt

```diff
@@ -29,6 +29,10 @@ Arrows
 
 Arrows: Core
 
+ * Added an arrow (transform_detected_object_set) for transforming detected
```

Comment:
Minor terminology nitpick here:  The "arrow" is usually what we call the plugin library.  So in this case you are adding an algorithm implementation to the existing "core" arrow.